dask_expr/tests/test_collection.py

```diff
@@ -681,6 +681,24 @@ def test_len(df, pdf):
     assert isinstance(expr.Lengths(df2.expr).optimize(), expr.Literal)
 
 
+def test_astype_simplify(df, pdf):
+    q = df.astype({"x": "float64", "y": "float64"})["x"]
+    result = optimize(q, fuse=False)
+    expected = df["x"].astype({"x": "float64"})
+    assert result._name == expected._name
+    assert_eq(q, pdf.astype({"x": "float64", "y": "float64"})["x"])
+
+    q = df.astype({"y": "float64"})["x"]
+    result = optimize(q, fuse=False)
+    expected = df["x"]
+    assert result._name == expected._name
+
+    q = df.astype("float64")["x"]
+    result = optimize(q, fuse=False)
```

Comment:
If you ever want, this is the same as `q.simplify()`

Today `optimize` calls `simplify` then `fuse`.  Sometimes just calling simplify can be simpler than `optimize(..., fuse=False)`

Reply:
I wasn't sure whether this might change at some point, so I decided to go with optimize hoping that this might be more stable (not sure if this is ever relevant though)

Reply:
I could also imagine the `fuse=` keyword going away.  In general I don't think that much is stable :)

Reply:
:)